rayon-core/src/scope/mod.rs

```diff
@@ -15,6 +15,8 @@ use unwind;
 #[cfg(test)]
 mod test;
 
+/// Represents a fork-join scope which can be used to spawn any number 
+/// of tasks. See [`Scope()`] for more information.
```

Comment:
Ah.  I'm very unfamiliar with whatever parsing engine you are planning to use for doc generation, so this is a brand new experience.  I'll give it another shot!